src/source/CppGenerator.scala

```diff
@@ -66,7 +66,7 @@ class CppGenerator(spec: Spec) extends Generator(spec) {
       }
     },
     w => {
-      // std::hash specialization has to go *outside* of the wrapNs
+      // std::hash specialization has to go *outside* of the wraps
```

Comment:
I think this was correct before - `wrapNs` puts a namespace declaration around things, which we don't want here.